<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="gbibc-dv00-17-v" name="com.ics.tests.ia.RegressionCoreIA" tests="46" failures="1" timestamp="5 Sep 2017 14:30:18 GMT" time="960.098" errors="4">
  <testcase name="testcase52053" time="13.845" classname="com.ics.tests.ia.RegressionCoreIA"/>
  <testcase name="testcase54145" time="12.617" classname="com.ics.tests.ia.RegressionCoreIA"/>
  <testcase name="testcase52054" time="21.407" classname="com.ics.tests.ia.RegressionCoreIA"/>
  <testcase name="testcase52089" time="12.878" classname="com.ics.tests.ia.RegressionCoreIA"/>
  <testcase name="testcase52094" time="9.456" classname="com.ics.tests.ia.RegressionCoreIA"/>
  <testcase name="testcase53480" time="2.108" classname="com.ics.tests.ia.RegressionCoreIA"/>
  <testcase name="testcase54138" time="12.152" classname="com.ics.tests.ia.RegressionCoreIA"/>
  <testcase name="testcase77820" time="53.289" classname="com.ics.tests.ia.RegressionCoreIA"/>
  <testcase name="testcase54350" time="2.289" classname="com.ics.tests.ia.RegressionCoreIA">
    <error type="org.openqa.selenium.NoSuchElementException" message="Unable to find element with xpath == //table[@id=&#039;dtMainQuery_grid&#039;]/tbody/tr/td[8] (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 287 milliseconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: &#039;2.48.2&#039;, revision: &#039;41bccdd10cf2c0560f637404c2d96164b67d9d67&#039;, time: &#039;2015-10-09 13:08:06&#039;
System info: host: &#039;gbibc-dv00-17-v&#039;, ip: &#039;129.227.32.254&#039;, os.name: &#039;Windows Server 2012 R2&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;6.3&#039;, java.version: &#039;1.8.0_121&#039;
Driver info: org.openqa.selenium.ie.InternetExplorerDriver
Capabilities [{browserAttachTimeout=0, enablePersistentHover=true, ie.forceCreateProcessApi=false, pageLoadStrategy=normal, ie.usePerProcessProxy=false, ignoreZoomSetting=true, handlesAlerts=true, version=11, platform=WINDOWS, nativeEvents=true, ie.ensureCleanSession=false, elementScrollBehavior=0, ie.browserCommandLineSwitches=, requireWindowFocus=false, browserName=internet explorer, initialBrowserUrl=http://localhost:5978/, takesScreenshot=true, javascriptEnabled=true, ignoreProtectedModeSettings=false, enableElementCacheCleanup=true, cssSelectorsEnabled=true, unexpectedAlertBehaviour=dismiss}]
Session ID: c6e7b7e2-bce7-4342-b5d4-7c36ce7c9d23
*** Element info: {Using=xpath, value=//table[@id=&#039;dtMainQuery_grid&#039;]/tbody/tr/td[8]}">
      <![CDATA[org.openqa.selenium.NoSuchElementException: Unable to find element with xpath == //table[@id='dtMainQuery_grid']/tbody/tr/td[8] (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 287 milliseconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.48.2', revision: '41bccdd10cf2c0560f637404c2d96164b67d9d67', time: '2015-10-09 13:08:06'
System info: host: 'gbibc-dv00-17-v', ip: '129.227.32.254', os.name: 'Windows Server 2012 R2', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_121'
Driver info: org.openqa.selenium.ie.InternetExplorerDriver
Capabilities [{browserAttachTimeout=0, enablePersistentHover=true, ie.forceCreateProcessApi=false, pageLoadStrategy=normal, ie.usePerProcessProxy=false, ignoreZoomSetting=true, handlesAlerts=true, version=11, platform=WINDOWS, nativeEvents=true, ie.ensureCleanSession=false, elementScrollBehavior=0, ie.browserCommandLineSwitches=, requireWindowFocus=false, browserName=internet explorer, initialBrowserUrl=http://localhost:5978/, takesScreenshot=true, javascriptEnabled=true, ignoreProtectedModeSettings=false, enableElementCacheCleanup=true, cssSelectorsEnabled=true, unexpectedAlertBehaviour=dismiss}]
Session ID: c6e7b7e2-bce7-4342-b5d4-7c36ce7c9d23
*** Element info: {Using=xpath, value=//table[@id='dtMainQuery_grid']/tbody/tr/td[8]}
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:206)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:158)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:647)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:353)
	at org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:490)
	at org.openqa.selenium.By$ByXPath.findElement(By.java:361)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:345)
	at org.openqa.selenium.support.pagefactory.DefaultElementLocator.findElement(DefaultElementLocator.java:69)
	at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:38)
	at com.sun.proxy.$Proxy11.getWrappedElement(Unknown Source)
	at org.openqa.selenium.remote.internal.WebElementToJsonConverter.apply(WebElementToJsonConverter.java:49)
	at com.google.common.collect.Iterators$8.transform(Iterators.java:794)
	at com.google.common.collect.TransformedIterator.next(TransformedIterator.java:48)
	at com.google.common.collect.Iterators.addAll(Iterators.java:357)
	at com.google.common.collect.Lists.newArrayList(Lists.java:147)
	at com.google.common.collect.Lists.newArrayList(Lists.java:129)
	at org.openqa.selenium.remote.RemoteWebDriver.executeScript(RemoteWebDriver.java:554)
	at com.ics.ia.pages.iaSearchItemPage.validateFrontAndRearImage(iaSearchItemPage.java:2397)
	at com.ics.tests.ia.RegressionCoreIA.testcase54350(RegressionCoreIA.java:511)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
    </error>
  </testcase> <!-- testcase54350 -->
  <testcase name="testcase61028" time="10.660" classname="com.ics.tests.ia.RegressionCoreIA"/>
  <testcase name="testcase52078" time="7.476" classname="com.ics.tests.ia.RegressionCoreIA">
    <error type="org.openqa.selenium.UnhandledAlertException" message="Modal dialog present: The webpage you are viewing is trying to close the tab.

Do you want to close this tab?
Build info: version: &#039;2.48.2&#039;, revision: &#039;41bccdd10cf2c0560f637404c2d96164b67d9d67&#039;, time: &#039;2015-10-09 13:08:06&#039;
System info: host: &#039;gbibc-dv00-17-v&#039;, ip: &#039;129.227.32.254&#039;, os.name: &#039;Windows Server 2012 R2&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;6.3&#039;, java.version: &#039;1.8.0_121&#039;
Driver info: org.openqa.selenium.ie.InternetExplorerDriver
Capabilities [{browserAttachTimeout=0, enablePersistentHover=true, ie.forceCreateProcessApi=false, pageLoadStrategy=normal, ie.usePerProcessProxy=false, ignoreZoomSetting=true, handlesAlerts=true, version=11, platform=WINDOWS, nativeEvents=true, ie.ensureCleanSession=false, elementScrollBehavior=0, ie.browserCommandLineSwitches=, requireWindowFocus=false, browserName=internet explorer, initialBrowserUrl=http://localhost:5978/, takesScreenshot=true, javascriptEnabled=true, ignoreProtectedModeSettings=false, enableElementCacheCleanup=true, cssSelectorsEnabled=true, unexpectedAlertBehaviour=dismiss}]
Session ID: c6e7b7e2-bce7-4342-b5d4-7c36ce7c9d23
*** Element info: {Using=xpath, value=//ul[@class=&#039;dropdown-menu&#039;]/li/a}">
      <![CDATA[org.openqa.selenium.UnhandledAlertException: Modal dialog present: The webpage you are viewing is trying to close the tab.

Do you want to close this tab?
Build info: version: '2.48.2', revision: '41bccdd10cf2c0560f637404c2d96164b67d9d67', time: '2015-10-09 13:08:06'
System info: host: 'gbibc-dv00-17-v', ip: '129.227.32.254', os.name: 'Windows Server 2012 R2', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_121'
Driver info: org.openqa.selenium.ie.InternetExplorerDriver
Capabilities [{browserAttachTimeout=0, enablePersistentHover=true, ie.forceCreateProcessApi=false, pageLoadStrategy=normal, ie.usePerProcessProxy=false, ignoreZoomSetting=true, handlesAlerts=true, version=11, platform=WINDOWS, nativeEvents=true, ie.ensureCleanSession=false, elementScrollBehavior=0, ie.browserCommandLineSwitches=, requireWindowFocus=false, browserName=internet explorer, initialBrowserUrl=http://localhost:5978/, takesScreenshot=true, javascriptEnabled=true, ignoreProtectedModeSettings=false, enableElementCacheCleanup=true, cssSelectorsEnabled=true, unexpectedAlertBehaviour=dismiss}]
Session ID: c6e7b7e2-bce7-4342-b5d4-7c36ce7c9d23
*** Element info: {Using=xpath, value=//ul[@class='dropdown-menu']/li/a}
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:206)
	at org.openqa.selenium.remote.ErrorHandler.createUnhandledAlertException(ErrorHandler.java:187)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:154)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:647)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:353)
	at org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:490)
	at org.openqa.selenium.By$ByXPath.findElement(By.java:361)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:345)
	at org.openqa.selenium.support.pagefactory.DefaultElementLocator.findElement(DefaultElementLocator.java:69)
	at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:38)
	at com.sun.proxy.$Proxy11.isDisplayed(Unknown Source)
	at com.ics.ia.pages.iaSearchItemPage.validateLogout(iaSearchItemPage.java:2234)
	at com.ics.tests.ia.RegressionCoreIA.testcase52078(RegressionCoreIA.java:794)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
    </error>
  </testcase> <!-- testcase52078 -->
  <testcase name="testcase127466" time="0.194" classname="com.ics.tests.ia.RegressionCoreIA"/>
  <testcase name="testcase120425" time="12.944" classname="com.ics.tests.ia.RegressionCoreIA"/>
  <testcase name="testcase127467" time="3.179" classname="com.ics.tests.ia.RegressionCoreIA"/>
  <testcase name="testcase77834" time="47.010" classname="com.ics.tests.ia.RegressionCoreIA"/>
  <testcase name="testcase52070" time="20.154" classname="com.ics.tests.ia.RegressionCoreIA"/>
  <testcase name="testcase53498" time="14.863" classname="com.ics.tests.ia.RegressionCoreIA">
    <error type="java.lang.StringIndexOutOfBoundsException" message="String index out of range: 19">
      <![CDATA[java.lang.StringIndexOutOfBoundsException: String index out of range: 19
	at java.lang.String.substring(Unknown Source)
	at com.ics.ia.pages.iaSearchItemPage.validateMinimumSearchCriteria(iaSearchItemPage.java:1476)
	at com.ics.tests.ia.RegressionCoreIA.testcase53498(RegressionCoreIA.java:256)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
    </error>
  </testcase> <!-- testcase53498 -->
  <testcase name="testcase53482" time="4.161" classname="com.ics.tests.ia.RegressionCoreIA"/>
  <testcase name="testcase54353" time="12.230" classname="com.ics.tests.ia.RegressionCoreIA"/>
  <testcase name="testcase77818" time="33.237" classname="com.ics.tests.ia.RegressionCoreIA"/>
  <testcase name="testcase77836" time="91.421" classname="com.ics.tests.ia.RegressionCoreIA"/>
  <testcase name="testcase52068" time="29.056" classname="com.ics.tests.ia.RegressionCoreIA"/>
  <testcase name="testcase60964" time="1.605" classname="com.ics.tests.ia.RegressionCoreIA">
    <failure type="java.lang.AssertionError" message="Cursor is NOT placed in &#039;Processing Date&#039; Field expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Cursor is NOT placed in 'Processing Date' Field expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at com.ics.seleniumCoreUtilis.Component.assertTrue(Component.java:122)
	at com.ics.ia.pages.iaSearchItemPage.validateProcDateFocus(iaSearchItemPage.java:1219)
	at com.ics.tests.ia.RegressionCoreIA.testcase60964(RegressionCoreIA.java:120)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
    </failure>
  </testcase> <!-- testcase60964 -->
  <testcase name="testcase54539" time="26.668" classname="com.ics.tests.ia.RegressionCoreIA"/>
  <testcase name="testcase52060" time="19.021" classname="com.ics.tests.ia.RegressionCoreIA"/>
  <testcase name="testcase54755" time="6.191" classname="com.ics.tests.ia.RegressionCoreIA"/>
  <testcase name="testcase127470" time="5.400" classname="com.ics.tests.ia.RegressionCoreIA"/>
  <testcase name="testcase54149" time="11.283" classname="com.ics.tests.ia.RegressionCoreIA"/>
  <testcase name="testcase53660" time="3.366" classname="com.ics.tests.ia.RegressionCoreIA"/>
  <testcase name="testcase120495" time="0.596" classname="com.ics.tests.ia.RegressionCoreIA"/>
  <testcase name="testcase59567" time="23.845" classname="com.ics.tests.ia.RegressionCoreIA"/>
  <testcase name="testcase131482" time="0.281" classname="com.ics.tests.ia.RegressionCoreIA">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.ics.ia.pages.iaSearchItemPage.validateOutboxMenu(iaSearchItemPage.java:1356)
	at com.ics.tests.ia.RegressionCoreIA.testcase131482(RegressionCoreIA.java:780)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
    </error>
  </testcase> <!-- testcase131482 -->
  <testcase name="testcase54148" time="10.057" classname="com.ics.tests.ia.RegressionCoreIA"/>
  <testcase name="testcase54559" time="16.909" classname="com.ics.tests.ia.RegressionCoreIA"/>
  <testcase name="testcase54177" time="224.884" classname="com.ics.tests.ia.RegressionCoreIA"/>
  <testcase name="testcase54557" time="16.366" classname="com.ics.tests.ia.RegressionCoreIA"/>
  <testcase name="testcase54333" time="20.915" classname="com.ics.tests.ia.RegressionCoreIA"/>
  <testcase name="loadDataToArchive1" time="17.863" classname="com.ics.tests.ia.RegressionCoreIA"/>
  <testcase name="testcase54754" time="5.367" classname="com.ics.tests.ia.RegressionCoreIA"/>
  <testcase name="testcase54140" time="8.206" classname="com.ics.tests.ia.RegressionCoreIA"/>
  <testcase name="testcase54139" time="15.317" classname="com.ics.tests.ia.RegressionCoreIA"/>
  <testcase name="testcase53483" time="4.800" classname="com.ics.tests.ia.RegressionCoreIA"/>
  <testcase name="testcase54981" time="26.956" classname="com.ics.tests.ia.RegressionCoreIA"/>
  <testcase name="testcase52088" time="26.454" classname="com.ics.tests.ia.RegressionCoreIA"/>
  <testcase name="testcase127469" time="19.475" classname="com.ics.tests.ia.RegressionCoreIA"/>
  <testcase name="testcase52061" time="21.647" classname="com.ics.tests.ia.RegressionCoreIA"/>
</testsuite> <!-- com.ics.tests.ia.RegressionCoreIA -->
